HLTrigger/special/python/hltppsJetComparisonFilter_cfi.py

```diff
@@ -0,0 +1,8 @@
+from Validation.CTPPS.hltppsJetComparisonFilterDefault_cfi import HLTPPSJetComparisonFilterDefault as _HLTPPSJetComparisonFilterDefault
+HLTPPSJetComparisonFilter = _HLTPPSJetComparisonFilterDefault.clone()
+
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(HLTPPSJetComparisonFilter, useNewLHCInfo = True)
+
+from Configuration.Eras.Modifier_ctpps_directSim_cff import ctpps_directSim
+ctpps_directSim.toModify(HLTPPSJetComparisonFilter, useNewLHCInfo = False)
```

Comment:
what's the purpose of this file?

Reply:
sorry, where is this file referenced elsewhere in cmssw?

Reply:
I don't know, but if `hltppsJetComparisonFilter_cfi.py` is used anywhere in the code the one I created should override it